packages/core/src/lib/waku_message/version_0.ts

```diff
@@ -1,7 +1,7 @@
 import type {
-  DecodedMessage,
-  Decoder,
-  Encoder,
+  DecodedMessage as IDecodedMessage,
+  Decoder as IDecoder,
+  Encoder as IEncoder,
```

Comment:
Can we just make this change a more permanent change?
We should define the interface is `IInterfaceName` and type as `TTypeName` and enum as `EEnumName`, instead of _importing as_